modules/ROOT/pages/streaming-about.adoc

```diff
@@ -271,6 +248,17 @@ increments at 100 objects per increment and allows a maximum buffer size of
  </sfdc:query>
 ----
 
+[[disable_repeatable_streaming]]
+== Disabling Repeatable Streaming
+
+You can disable repeatable streaming through the `non-repeatable-stream` and `non-repeatable-iterable` strategies. The strategy to use depends on the type of stream.
```

Comment:
Should we clarify what we mean by type of stream? I don't see that in the following text.

Reply:
type of stream is covered in preceding (stream of objects or bytes)